### PR TITLE
Misc. INDEL realigner bugfixes

### PR DESCRIPTION
**Pull request by fnothaft**

### CODE REVIEW

File: adam-core/src/main/scala/org/bdgenomics/adam/rdd/read/realignment/RealignIndels.scala [hunk 1]
```diff
@@ -359,23 +360,41 @@ private[read] class RealignIndels(
...
-            } else
+              try {
+                builder.setMismatchingPositions(MdTag.moveAlignment(r, newCigar, reference.drop(remapping), refStart + remapping).toString())
+                builder.setOldPosition(r.getStart())
+                builder.setOldCigar(r.getCigar())
+                builder.setCigar(newCigar.toString)
+                new RichAlignmentRecord(builder.build())
+              } catch {
+                case t: Throwable => {
+                  println("Caught %s when trying to move alignment to %s for %s.".format(
```
Comment (heuermh) [inline adam-core/src/main/scala/org/bdgenomics/adam/rdd/read/realignment/RealignIndels.scala]: use Logger here?

**Comment by heuermh** [comment]
@fnothaft ping for logging fix

**Comment by fnothaft** [comment]
Fixed logging and rebased.

**Comment by heuermh** [comment]
Thank you, @fnothaft

**Pull request merged**